src/elements/stepper/stepper/stepper.stories.ts

```diff
@@ -0,0 +1,445 @@
+import { withActions } from '@storybook/addon-actions/decorator';
```

Comment:
Minor improvement: I see some duplicated code here in templates, which can be refactored.